### ISSUE TITLE
Better dark mode color palette

### ISSUE THREAD

**Issue by Josh-Cena** [facebookincubator/infima#199]
Our dark mode background is too dark. While it's fine for the Docusaurus website because the text is relatively sparse, as soon as the text gets dense, it literally hurts my eyes.

![image](https://user-images.githubusercontent.com/55398995/148635772-d7db2aa8-1799-4dda-babc-8b858a16079d.png)

Not only is the background too black, but the admonition is too blue as well. The light mode contrast seems so much better.

GitHub and Twitter also have this problem (hence I never read long READMEs on GitHub), but VuePress's palette is very nice:

![image](https://user-images.githubusercontent.com/55398995/148635813-f754cf49-e9fc-4488-b66c-3d53af48c449.png)

**Comment by lex111** [comment]
Hmm, I don't have that much inconveniences when reading, but maybe we should darken foreground text color because it's probably bright a bit now. Just compare and let me know what works best for you:

| Before   | After    |
| -------- | -------- |
| ![image](https://user-images.githubusercontent.com/4408379/153592078-4e260661-531a-492f-9e32-e68281d445f2.png) | ![image](https://user-images.githubusercontent.com/4408379/153593416-c0a85f8b-77dd-46d1-95ab-df87bf708d40.png) |

I actually like the color palette of Microsoft Docs, where the background is even darker than in Docusaurus. Wonder if you are comfortable reading their docs https://docs.microsoft.com/en-us/aspnet/core/introduction-to-aspnet-core?view=aspnetcore-6.0

**Comment by Josh-Cena** [comment]
Ah, I think their docs are good for me as well. I don't know what's the key here now 😄

### Less eye strain 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55398995/153595130-56c618d3-d3b9-48e3-ac73-2fa43bb4d6b5.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55398995/153595257-bd30fc2d-ce8a-424c-a214-cc65e18bd43e.png">

<img width="865" alt="image" src="https://user-images.githubusercontent.com/55398995/153595449-0f3e5d5b-b5e9-44cd-9422-be63192e501e.png">

### More eye strain

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55398995/153595485-41be4d6a-e426-46b7-a4c1-b5f9e424f8d7.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55398995/153595714-0156af99-44b9-43c7-8b51-510f7e18df14.png">

It may be a combo of font, text density, and background/foreground contrast?

**Comment by lex111** [comment]
Maybe so, although I would definitely tweak text color for dark mode. As another example, the new Vue.js site looks good in enabled dark mode: https://vuejs.org/guide/extras/ways-of-using-vue.html

**Comment by Josh-Cena** [comment]
Yes, whatever makes it looks better, may not necessarily be the background 👍

**Comment by slorber** [comment]
Hey, also missed this issue.

I also find it uncomfortable to read a Docusaurus site in dark mode. For me the text is too bright.

In light mode it's fine right?

Would you agree on this change only in dark mode?

```diff
-    --ifm-color-content: var(--ifm-color-emphasis-900);
+   --ifm-color-content: var(--ifm-color-emphasis-700);
```

Not sure of the exact value to use, and it doesn't even fix the issue in all places though 😅 

Here's what -300 do, the sidebar/toc/admonitions remain too bright

<img width="1914" alt="image" src="https://user-images.githubusercontent.com/749374/157494311-ecfa99f6-1d27-4458-a850-6a8f17216e6c.png">

**Comment by pixelass** [comment]
I added a comment here that might help: https://github.com/facebookincubator/infima/issues/139#issuecomment-2093539041
